src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -36,11 +38,12 @@ const CheckBoxCustom = (props: {value: boolean; label: string; disabled?: boolea
           backgroundColor: COLORS.white,
           marginRight: 5
         }}>
+        {/* TODO: Garry, masih ada warna hitam kalau disabled */}
         <CheckBox
           disabled={props.disabled || false}
           value={props.value}
-          onCheckColor={COLORS.white}
-          tintColors={{true: COLORS.signed_primary, false: COLORS.white}}
+          onCheckColor={COLORS.gray100}
+          tintColors={{true: COLORS.signed_primary, false: COLORS.gray100}}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comment on line 41 indicates that there's an issue with the color when the CheckBox component is disabled. If this issue has been resolved with the changes made in this hunk, please remove the comment. If not, consider addressing it before merging.



<!-- This is an auto-generated comment by OSS CodeRabbit -->